<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LC_DrawBoundingBoxOptions</class>
 <widget class="QWidget" name="LC_DrawBoundingBoxOptions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>22</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item row="0" column="2">
    <widget class="QCheckBox" name="cbAsGroup">
     <property name="toolTip">
      <string>If checked, bounding box for the entire selection will be created. Otherwise, it will be created for individual entities. </string>
     </property>
     <property name="text">
      <string>Selection As Group</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QCheckBox" name="cbCornerPointsOnly">
     <property name="toolTip">
      <string>If selected, only corner points will be drawn for bounding box. Otherwise, edge lines will be drawn.</string>
     </property>
     <property name="text">
      <string>Corner points only</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QCheckBox" name="cbPolyline">
     <property name="toolTip">
      <string>If selected, bounding box is drawn as polyline</string>
     </property>
     <property name="toolTipDuration">
      <number>4</number>
     </property>
     <property name="text">
      <string>Polyline</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
